src/zenml/orchestrators/step_runner.py

```diff
@@ -421,6 +421,21 @@ def _load_input_artifact(
         """
         # Skip materialization for `UnmaterializedArtifact`.
         if data_type == UnmaterializedArtifact:
+            # >>> this block solves tests/unit/orchestrators/test_step_runner.py::test_loading_unmaterialized_input_artifact
+            # - pydantic.errors.ConfigError: field "user" not yet prepared so type is still a ForwardRef, you might
+            # need to call UnmaterializedArtifact.update_forward_refs().
+            from zenml.models import (
+                RunMetadataResponseModel,
+                UserResponseModel,
+                WorkspaceResponseModel,
+            )
+
+            UnmaterializedArtifact.update_forward_refs(
+                UserResponseModel=UserResponseModel,
+                WorkspaceResponseModel=WorkspaceResponseModel,
+                RunMetadataResponseModel=RunMetadataResponseModel,
+            )
+            # >>> end of workaround block
```

Comment:
I have no clue how to solve it differently - I spent too much time trying to fight this tiny thing today. Decided to unblock myself, since the timeline is tight. Any ideas are welcome!

Reply:
Moved `UnmaterializedArtifact` into `zenml.artifacts` to fix this now. However, this is a breaking change since some users might currently import it from the old place. I'll add it to the description of the big PR so we don't forget that.